worker/runc/runc_test.go

```diff
@@ -116,6 +115,14 @@ func TestRuncWorker(t *testing.T) {
 	err = w.Executor.Exec(ctx, meta, root, nil, nil, nil, nil)
 	require.NoError(t, err)
 
+	meta = executor.Meta{
+		Args: []string{"/bin/sh", "-c", "ls", "/etc/resolv.conf"},
```

Comment:
`Args: []string{"ls", "/etc/resolv.conf"}`

Reply:
Looks like `/etc/resolv.conf` does not exist and `ls /etc/resolv.conf` failing.

Note that `{"/bin/sh", "-c", "ls", "/etc/resolv.conf"}` executes `ls`, not `ls /etc/resolv.conf`.

Reply:
Or maybe `/bin/ls` instead of `ls` if there is PATH issue

Reply:
It was path issue. Fixed